engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/sql/columnmodel/CreateColumnModel.scala

```diff
@@ -79,7 +80,7 @@ object CreateColumnModel {
              J_BOOLEAN =>
           Some(Bool)
           //TODO: other date types?
-        case DATE | LOCAL_DATE_TIME =>
+        case DATE | LOCAL_DATE_TIME | TIMESTAMP =>
```

Comment:
I added this to support use of TIMESTAMP type columns, we have some of those in our project. Do you think it's the right way to handle it?